opencolorio_config_aces/clf/transforms/canon/generate.py

```diff
@@ -0,0 +1,141 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+"""
+*Canon* CLF Transforms Generation
+===============================
```

Comment:
**Minor**

*reStructuredText* tolerates longer underlines but not the shorter ones!